src/pages/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import WishlistCard from "../../components/Card/WishlistCard";
+import { useWishlist } from "../../context/wishlist-context";
+import "./wishlist.css";
+import emptyWishlist from "../../assets/empty-wishlist.png";
+
+const Wishlist = () => {
+	const { wishlistProducts } = useWishlist();
+	console.log(wishlistProducts);
+	const { removeFromWishlist } = useWishlist();
+	return (
+		<>
+			<div class="wishlist-product-container">
+				<div class="wishlist-title">
+					My Wishlist
+					<span class="wishlist-total">
+						({wishlistProducts.length} Items added)
+					</span>
+				</div>
+
+				{wishlistProducts.length < 1 && (
+					<div className="empty-wishlist-container">
+						{/* <img src={emptyWishlist} /> */}
+						<h1>🛍️</h1>
+						<p>There is nothing in your wishlist, Let's add something!</p>'
+						<Link to="/products">
+							<button className=" btn btn-secondary btn-dark">
+								Add Items to Wishlist
+							</button>
+						</Link>
+					</div>
+				)}
+
+				{/* <!-- Wishlised products --> */}
+				<div class="wishlisted-products">
+					{wishlistProducts &&
+						wishlistProducts.map(
+							({
```

Comment:
- here can we pass the whole ```productDetails``` object we are getting on each iteration and destructure its properties inside ```WishlistCard``` component itself instead of passing multiple individual props.

- It will make code more readable and will make your life easier.